packages/core/src/components/tooltip/tooltip.tsx

```diff
@@ -114,13 +96,29 @@ export class Tooltip<
         this.popoverRef.current?.reposition();
     }
 
+    protected validateProps(props: TooltipProps<T>) {
+        const childrenCount = React.Children.count(props.children);
+        if (childrenCount > 1) {
+            console.warn(Errors.POPOVER_WARN_TOO_MANY_CHILDREN);
```

Comment:
why repeat this warning here but not the others?

Reply:
All the others are caught within `Popover`'s `validateProps`. This one needs to be checked here though because we wrap any passed `children` in the `Utils.ensureElement`. So if we didn't have this `validateProps`, `Popover`'s would never warn for multiple children.

Reply:
thanks for the explanation!